src/app/Instance/BridgeOverview/Components/BODashboardTableView.tsx

```diff
@@ -128,7 +129,22 @@ export const BODashboardTableView = (
                 </Stack>
               </Td>
               <Td isActionCell>
-                {rowActions ? <ActionsColumn items={rowActions} /> : null}
+                {
+                  <ActionsColumn
+                    items={[
+                      {
+                        title: "Edit",
+                        isDisabled: !canEditResource(item.status),
+                        onClick: () => onEditItem(item.id),
+                      },
+                      {
+                        title: "Delete",
+                        isDisabled: !item || !canDeleteResource(item.status),
```

Comment:
I can see the result of this condition in our storybook. I am not sure if we could create the component test to check enabled/disabled actions right now. This component has not got any component test.